mne/tests/test_epochs.py

```diff
@@ -1477,17 +1477,20 @@ def test_epochs_io_preload(tmp_path, preload):
 
 @pytest.fixture(scope="session")
 def epochs_factory():
-    """Function to create fake Epochs object."""  # noqa: D401 (imperative mood)
+    """Function to create fake Epochs object.
+
+    Metadata and concat address gh-5102, gh-7897.
+
+    """  # noqa: D401 (imperative mood)
```

Comment:
```suggestion
    """Create fake Epochs object.

    Metadata and concat address gh-5102, gh-7897.
    """
```
The linter is right to complain and it's less work (and code) to make it happy than it is to ignore it here